src/evo/dmnstate.h

```diff
@@ -52,16 +99,42 @@ class CDeterministicMNState
     CScript scriptPayout;
     CScript scriptOperatorPayout;
 
+    uint160 platformNodeID{};
```

Comment:
no validation on this?

Reply:
@PastaPastaPasta What do you mean by validation here?

Reply:
Well, shouldn't the node be invalid for a null value? In order to become valid the operator will need to update service?

Reply:
@PastaPastaPasta Not at this point (`CDeterministicMNState`). Validations are already made when check ProRegTx and ProUpServTx